lib/widgets-page.php

```diff
@@ -28,15 +28,6 @@ class="blocks-widgets-container"
  * @param string $hook Page.
  */
 function gutenberg_widgets_init( $hook ) {
-	if ( 'widgets.php' === $hook ) {
-		wp_enqueue_style( 'wp-block-library' );
-		wp_enqueue_style( 'wp-block-library-theme' );
-		wp_add_inline_style(
-			'wp-block-library-theme',
-			'.wp-block-widget-textarea { width: 100%; min-height: 5em; margin: 8px 0 16px 0; }'
```

Comment:
The wp-block-widget-textarea style is probably needed as we are still outputting a text area for blocks in lib/class-wp-widget-block.php. But not sure if here is the best place to output these styles I guess given that these styles are small we could add them as inline styles to lib/class-wp-widget-block.php?

Reply:
If we wrap the `<textarea>` in a `<p>` and swap `class="widefat text wp-block-widget-textarea"` for `class="large-text code"` then it should look right.

Reply:
I made some changes in c0ac7db:

- Wrapped the `textarea` in a `<p>` like other widget fields
- Added a label, which seemed to be missing
- Removed the `text` class and added the `code` class.

<img width="450" alt="Screenshot 2021-05-04 at 10 38 11 am" src="https://user-images.githubusercontent.com/677833/116955490-4c165980-acc5-11eb-828e-549c0fab40d3.png">